### PR TITLE
Enum fields

### PR DESCRIPTION
**Pull request by JElgar**
Return enums as objects, with value and description

### CODE REVIEW

File: uobtheatre/utils/schema.py [hunk 1]
```diff
@@ -7,21 +7,68 @@
...
+                description=getattr(cls, f"get_{field_name}_display")(),
+            )
+
+        return resolver
+
+    @classmethod
+    def __init_subclass_with_meta__(self, *args, **kwargs):
+        """
+        Overwrite the DjangoObjectType init to add in resolvers for every
+        GrapheneImageField.
```
Comment (JElgar) [inline uobtheatre/utils/schema.py:33]:
```suggestion
        EnumNode.
```

**Comment by JElgar** [comment]
@atoff @streth11 opinions?

File: uobtheatre/bookings/test/test_schema.py [hunk 2]
```diff
@@ -1218,11 +1229,15 @@ def test_pay_booking_success(mock_square, gql_client_flexible, gql_id):
     mock_square.success = True
 
     response = client.execute(request_query % gql_id(booking.id, "BookingNode"))
+    print(response)
```
Comment (atoff) [inline uobtheatre/bookings/test/test_schema.py:1232]: Is this meant to be left in 🤔

**Review by atoff — COMMENTED** [review]
Yeah this was pretty much what I was thinking of (going to be fun to implement in mirage 🥳)

File: uobtheatre/bookings/test/test_schema.py (hunk 2 above)
Comment (JElgar) [reply uobtheatre/bookings/test/test_schema.py:1232]: Heroic 
```suggestion
```

**Review by atoff — APPROVED** [review]

**Pull request merged**